typings/@chainlink/index.d.ts

```diff
@@ -55,7 +55,8 @@ declare module '@chainlink/types' {
     apiKey?: string
     network?: string
     returnRejectedPromiseOnError?: Boolean
-    api: Partial<AxiosRequestConfig>
+    api?: Partial<AxiosRequestConfig>
+    threshold?: Record<string, number>
```

Comment:
Not sure if this is the ideal way to handle this, but I'm also not familiar with the decision to have a centralized Config type.

Reply:
Yeah, I feel your pain.

It's not meant as a centralized Config type, but as a base Config type where every adapter has a specific Config type that extends that base Config, and needs to implement a factory method:

```javascript
export const makeConfig = <C extends Config>(args...): C => {
```

But the infrastructure is not there yet.

Reply:
We should now be able to remove this line from here, as you defined a specific config for that specific adapter.